docker/cygnus-ngsi/Dockerfile

```diff
@@ -65,9 +67,9 @@ RUN yum -y install tar nc which git java-1.6.0-openjdk-devel && \
 # Create Cygnus log folder
 RUN mkdir ${FLUME_HOME}/log
 
-# Clone fiware-cygnus
-RUN git clone ${GIT_URL_CYGNUS} ${CYGNUS_HOME} && \
-    cd ${CYGNUS_HOME} && \
+# Instead of re-cloning, copy the fiware-cygnus repo, move there and checkout the propper branch
+COPY . ${CYGNUS_HOME}
```

Comment:
I updated the README.md with the right place where to run the command, and the command:

```
$ cd fiware-cygnus
$ docker build -t cygnus-ngsi -f docker/cygnus-ngsi/Dockerfile .
```

Of course, if you change the directory where the above command is run, it does not work.